_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: single
+sidebar:
+  nav: policies_bylaws.md
+title: Bylaws of the Living Journal of Computational Molecular Science
+excerpt: Bylaws of LiveCoMS.
+permalink: /policies/livecoms_bylaws/
+---
+
+1. Organization Title and Publication
+
+   The Journal shall be called the Living Journal of Computational
+   Molecular Science, hereafter referred to as LiveCoMS.
+
+   LiveCoMS shall produce one publication, the Living Journal of
+   Computational Molecular Science, with articles released on a rolling
+   basis online. LiveCoMS provides a peer-reviewed home for manuscripts
+   which share best practices in molecular modeling and simulation. These
+   works are living documents, regularly updated with community input,
+   and can include living or perpetual reviews, tutorials, comparisons
+   between software packages, and other documents which aim to improve
+   the studies in the field and require ongoing updates.
+
+1. Editorial Board
+
+   The editorial board shall consist of three Managing Editors, any
+   number of Associate Editors, and a Section Lead Editor for each type
+   of document published in LiveCoMS, and any number of Associate
+   Editors. Section Lead Editors may be either Managing Editors or
+   Associate Editors.
+
+   1. Responsibilities
+       1. The Managing Editors coordinate the operation of the Journal and
+          the finances of the journal, maintain the online presence.  Tasks
+          may be divided up among managing editors as they see fit, and specific
+          individual tasks may be assigned to individual associate editors.
```

Comment:
I'm trying to avoid having to make a list of all possible things that could be done (manage the payments to scholastica, raise money, recruit new editors, manage voting, etc, manage the email list).  I'd rather say "Whatever needs to be done, the managing editors will assign" rather than forget anything and then not have it clear who is in charge.

Reply:
I guess I'd perhaps just prefer a more concise wording like "The Managing Editors coordinate the operation of the Journal and the finances of the journal, maintain the online presence., etc., and delegate tasks to associate editors as appropriate." The "individual tasks may be assigned" part bugs me since it SOUNDS like it's giving some kind of a rule about how things work but it's actually not. :)

Reply:
Up for making a minor tweak along these lines? (Or I can do a PR to this PR, @mrshirts ).